torch/utils/cpp_extension.py

```diff
@@ -1714,6 +1754,8 @@ def _write_ninja_file(path,
     `library_target`: Name of the output library. Can be None; in that case,
                       we do no linking.
     `with_cuda`: If we should be compiling with CUDA.
+    `is_standalone`: If we should produce a standalone binary rather than a
```

Comment:
Actually, I don't quite understand the argument `is_standalone` here. What does `binary` in the description mean? An executable? Or an object file? If you refer to the latter one here, then how are you gonna use it? Does Python allow users to load the functions from an object file?

Reply:
If you refer to the former one, then the solution is simple. Just change the extension to `.exe`. And I think it will work out of box.